docs/rules/no-inline-assertions.md

```diff
@@ -0,0 +1,35 @@
+# Ensure assertions are not called from an inline function
+
+Translations: [Français](https://github.com/avajs/ava-docs/blob/master/fr_FR/related/eslint-plugin-ava/docs/rules/no-inline-assertions.md)
```

Comment:
BTW this link is a dummy french translation link. As I don't speak French (I wish I could), I hope I can connect to the people and translate this document after this PR gets merged.

Reply:
FYI @forresst

Reply:
@JLHwung Do not add this line yet. I will do the translation when this PR is merged. And I will add this link when the translation is done.

Reply:
Removed and thanks.